app/services/support_interface/applications_by_subject_route_and_degree_grade_export.rb

```diff
@@ -81,15 +81,15 @@ def grouping_key(hash)
 
     def choices_with_courses_and_subjects
       ApplicationChoice
-        .select('application_choices.id as id, application_choices.status as status, providers.provider_type as route, application_qualifications.grade as degree_class, application_form.id as application_form_id, application_form.phase as phase, courses.name as course_name, courses.level as course_level, ARRAY_AGG(subjects.name) as subject_names, ARRAY_AGG(subjects.code) as subject_codes, (CASE WHEN a2_latest_application_forms.candidate_id IS NOT NULL THEN true ELSE false END) AS is_latest_a2_app')
+        .select('application_choices.id as id, application_choices.status as status, providers.provider_type as route, application_qualifications.grade_hesa_code as degree_class, application_form.id as application_form_id, application_form.phase as phase, courses.name as course_name, courses.level as course_level, ARRAY_AGG(subjects.name) as subject_names, ARRAY_AGG(subjects.code) as subject_codes, (CASE WHEN a2_latest_application_forms.candidate_id IS NOT NULL THEN true ELSE false END) AS is_latest_a2_app')
```

Comment:
the HESA code will now be returned instead of the `degree_class`, could you update the docs found [here](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/exports/applications_by_subject_route_and_degree_grade_export.yml#L16) to indicate this is now a HESA code as well as rename `degree_class` here as it's no longer representative?